### PR TITLE
(DOCS-3826) Add APM naming convention

### PR DESCRIPTION
**Pull request by yoannrobin**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_collection/tracing_naming_convention/_index.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+| `http.request.content_length`               | `number` | The size of the request payload body in bytes                                                                                                                      |
+| `http.response.content_length`              | `number` | The size of the request payload body in bytes                                                                                                                      |
+| `http.request.content_length_uncompressed`  | `number` | The size of the uncompressed request payload body after transport decoding                                                                                         |
+| `http.response.content_length_uncompressed` | `number` | The size of the uncompressed response payload body after transport decoding                                                                                        |
+| `http.request.headers.*`                    | `string` | The request HTTP headers. None are collected by default but can optionally be configured with `DD_TRACE_HEADER_TAGS`.                                              |
```
Comment (brettlangdon) [inline content/en/tracing/trace_collection/tracing_naming_convention/_index.md:64]: We should link `DD_TRACE_HEADER_TAGS` to docs on how to use it.

File: content/en/tracing/trace_collection/tracing_naming_convention/_index.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+| `network.destination.transport` | `string` | Transport protocol used to make the outbound connection                   |
+
+### HTTP requests
+The following span tags can be used to describe HTTP client and server spans:
+
+| **Fullname**                                | **Type** | **Description**                                                                                                                                                    |
+|---------------------------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `http.status_code`                          | `string` | The HTTP response status code                                                                                                                                      |
+| `http.url`                                  | `string` | The URL of the HTTP request, including the obfuscated query string. For more informations on obfuscation refer to [Data security][2] |
```
Comment (brettlangdon) [inline content/en/tracing/trace_collection/tracing_naming_convention/_index.md:55]: we should document `http.url_details.*` tags as well.... once http url parsing is implemented on the backend

File: content/en/tracing/trace_collection/tracing_naming_convention/_index.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+| `network.host.ip`               | `string` | Local host IP address                                                     |
+| `network.client.port`           | `number` | The port of the client that initiated the connection                      |
+| `network.destination.port`      | `number` | Remote port number of the outbound connection                             |
+| `network.client.name`           | `string` | The hostname of the client that initiated the inbound connection          |
+| `network.destination.name`      | `string` | Remote hostname or similar where the outbound connection is being made to |
+| `network.host.name`             | `string` | Local hostname                                                            |
+| `network.client.transport`      | `string` | Transport protocol used to make the inbound connection                    |
+| `network.destination.transport` | `string` | Transport protocol used to make the outbound connection                   |
+
+### HTTP requests
```
Comment (brettlangdon) [inline content/en/tracing/trace_collection/tracing_naming_convention/_index.md:49]: missing `http.useragent` + `http.useragent_details.*`?

**Review by maycmlee — CHANGES_REQUESTED** [review]
Hey @yoannrobin, here's some feedback! I'll need to look into reformatting the HTTP table because it's overflowing: https://a.cl.ly/E0uyN1v9

**Review by maycmlee — APPROVED** [review]

**Pull request merged**